interface.html

```diff
@@ -293,6 +293,31 @@ <h4 class="panel-title" data-toggle="collapse" data-parent="#social-accordion" d
                       </div>
                     </div>
                   </div>
+                  <div class="form-group clearfix list-chat disabled hidden">
+                    <div class="col-sm-4 control-label">
+                      <label>Chat</label>
+                    </div>
+                    <div class="col-sm-8">
+                      <div class="checkbox checkbox-icon">
+                        <input type="checkbox" id="enable-chat">
+                        <label for="enable-chat">
+                          <span class="check"><i class="fa fa-check"></i></span> Allow users to start messaging directly from each entry
+                        </label>
+                        <div class="hidden-settings">
+                          <p>You need to create a chat screen to allow users to message each other. Select the screen where the chat feature will be found.</p>
+                          <div id="chat-link"></div>
+                          <div class="select-chat-email-holder">
+                            <label>Email field (Required)</label>
+                            <select name="select_chat_email" id="select_chat_email" class="form-control">
```

Comment:
Dropdowns should use https://developers.fliplet.com/UI-guidelines-interface.html#dropdown-list markup. To prevent UI inconsistency.

I realise that there's quite a lot of dropdown UI in this file that isn't using the markup linked above. Let's fix them too.